scripts/build.js

```diff
@@ -9,9 +9,20 @@ const buildPackage = require('./helpers/build');
 const exec = require('./helpers/bash');
 
 const directories = getDirectories(resolve(process.cwd(), 'packages'));
-const configs = directories.map(directory => readFileSync(resolve(`${directory}/lib`, 'package.json'), {
-	encoding: 'UTF-8'
-}));
+const configs = directories.map(directory => {
```

Comment:
Am I right that this code 'fixes' the wrong scaffold provided by our schematics? (part 1)

Reply:
Yep